packages/@datapull/atlassian-origin/index.js

```diff
@@ -20,96 +24,34 @@ class AtlassianOrigin {
       runner: this.pullData.bind(this)
     };
   }
-
+  // The purpose of this origin is to list all members of specific
+  // group in Atlassian Cloud. As at this moment Atlassian REST API
+  // doesn't have a proper end point to list all groups in specific
+  // organisation, to list all Atlassian users in specific organisation,
+  // that have access to jira tool, we will use this origin to pull
+  // all members of 'jira-software-users'. For those who have an access
+  // to confluence, we have to retrieve members of 'confluence-users' group.
```

Comment:
I would say this comment is specific to your usage of this datapull module and should not be here